coffee/base.py

```diff
@@ -151,7 +161,7 @@ def __deepcopy__(self, memo):
         return self.__class__(dcopy(self.children[0]), dcopy(self.children[1]))
 
     def gencode(self, not_scope=True, parent=None):
-        subtree = (" "+self.op+" ").join([n.gencode(not_scope, self) for n in self.children])
+        subtree = (" "+type(self).op+" ").join([n.gencode(not_scope) for n in self.children])
```

Comment:
Oh, I was just looking at the class property because it's a class property. I didn't realise I can access class properties directly on instances